google/cloud/bigtable/testing/table_integration_test.cc

```diff
@@ -78,15 +78,17 @@ std::string TableTestEnvironment::RandomTableId() {
   // proto files.
   constexpr int kMaxTableIdLength = 50;
   static std::string const prefix = "table-";
-  return CreateRandomId(prefix, kMaxTableIdLength - prefix.length());
+  return CreateRandomId(prefix,
```

Comment:
Suggestion: Perhaps consider an alternative where `CreateRandomId()` takes a `std::size_t max_length`?

Then it can be the one place that has to deal with `prefix` being too long, and we can probably avoid the reverse `std::size_t(n)` cast in `Sample()`.